walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +273,130 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
+
+  @Test
+  public void ignoreOtionalWithWholeDayMeeting() {
```

Comment:
typo: optional

Reply:
Usually the test name will be as follows: 

`givenState_expectedOutput`. 

So for instance, here I'd expect it to be
`optionalAttendeeWithWholeDayMeeting_ignored`

Reply:
That's interesting! They didn't use this in the tests that they wrote. should I adjust my tests to this style?

Reply:
Yes please:)

Reply:
done